### ISSUE TITLE
[QUESTION] SKIPPING ICMP DISCOVERY

### ISSUE THREAD

**Issue by Karmaz95** [RustScan/RustScan#426]
If I want to do host discovery via port scanning (not ICMP sweep and then port scanning on active hosts).
`nmap -Pn 10.10.10.0/24 -p-`

If I pass the `-Pn` as an arg, Rustscan will conduct a port scan over the whole subnet despite the host being inactive to ICMP packets?
`rustscan -a 10.10.110.0/24 --scan-order "Random" -- -Pn`
Or it will scan whole subnet with ICMP sweep and the parse the `-Pn` flag for nmap?

TL;DR;
I want to skip ICMP host discovery in Rustscan and jump straight to the port scanning, does Rustscan doing it always automagically or there is no option atm to do it?

Have a nice day,
Karol
